src/progressbar/progressbar.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('mgcrea.ngStrap.progressbar', [])
+.provider('$progressbar', function (){
+    var defaults = {
+      barType: '',
+      animate: function (){ return true;}
+    };
+
+    this.$get = function (){
+      return {
+        defaults: defaults
+      };
+    };
+  })
+  .directive('bsProgressbar', function ($progressbar){
+    return {
+      restrict: 'E',
+      transclude: true,
+      replace: true,
+      templateUrl: 'progressbar/progressbar.tpl.html',
+      scope:{
```

Comment:
AngularStrap uses `scope: true` accross the project, why use `&` for animate (and `@` for type)?